api/holiday_acres_api/views.py

```diff
@@ -21,6 +21,7 @@
 env = environ.Env()
 environ.Env.read_env(os.path.join(BASE_DIR, ".env"))
 user_service_var = env("USER_SERVICE")
+django_secret_key = env("SECRET_MIDDLE_KEY")
```

Comment:
Can you put `SECRET_MIDDLE_KEY` in the Django `.env.example`? https://github.com/mstephenhunt/holiday_acres/blob/0ac9d6befc4757aff7c9734cd4bb370eb6c093c4/api/.env.example

Reply:
Will do